modules/widgets/eu-cookie-law/form.php

```diff
@@ -13,6 +13,11 @@
 				/>
 				<?php esc_html_e( 'after the user clicks the dismiss button', 'jetpack' ); ?>
 			</label>
+			<?php if ( Jetpack::is_module_active( 'wordads' ) ) : ?>
+				<p>
+					<strong><em><?php esc_html_e( 'This option is strongly recommended for Ads users.', 'jetpack' ); ?></em></strong>
```

Comment:
Should we be a bit more specific and say "WordAds" (vs. just "Ads") here and maybe link to its support document to better explain the reference?

Reply:
Updated to "Jetpack Ads" in the most recent commit to make the reference more clear.